website/pages/downloads/style.module.css

```diff
@@ -1,61 +1,7 @@
 .root {
-  margin-top: 72px;
-
-  & a {
-    text-decoration: underline;
-  }
-
-  & h1 {
-    text-align: center;
-  }
-
-  --brand: var(--red);
-  --brand-d1: #c03e43;
+  --red-d1: #c03e43;
```

Comment:
Good call here. 

This is semi-fragile in terms of the component usage for the Boundary theme (as the component is dependent on this external declaration) -- but not really a real problem, because this is the only place where this will exist.

I have a task on my plate to clean up this theming, so I'll get to it eventually.